titus-server-master/src/main/java/com/netflix/titus/master/scheduler/SchedulerConfiguration.java

```diff
@@ -114,4 +115,16 @@
      */
     @DefaultValue("true")
     boolean isCriticalTierJobSpreadingEnabled();
+
+    /**
+     * Availability zone attribute name to look at on the {@link AgentInstance}.
+     */
+    @DefaultValue("availabilityZone")
+    String getAvailabilityZoneAttributeName();
```

Comment:
We should use standard names internally and map if needed at the connector layer.
This introduces unnecessary complexity.

Reply:
This is based on how the current model works. I agree that we should change the approach but I do not want to do it in this PR.